e2e-tests/functions

```diff
@@ -335,6 +344,26 @@ deploy_minio() {
         /usr/bin/aws --endpoint-url http://minio-service:9000 s3 mb s3://operator-testing'
 }
 
+deploy_chaos_mesh() {
+	desc 'install chaos-mesh'
+	helm del chaos-mesh --namespace chaos-testing || :
+	kubectl_bin delete ns chaos-testing || :
+	helm repo add chaos-mesh https://charts.chaos-mesh.org
+	kubectl_bin create ns chaos-testing
+	if version_gt "1.20"; then
+		helm install chaos-mesh chaos-mesh/chaos-mesh --namespace=chaos-testing --set chaosDaemon.runtime=containerd --set chaosDaemon.socketPath=/run/containerd/containerd.sock --set dashboard.create=false
```

Comment:
I believe we need to use fixed version here.

Reply:
Thanks! fixed the chaos-mesh chart version to `0.5.1`